content/blog/blog-post-06.md

```diff
@@ -0,0 +1,211 @@
+It will recur every 3rd Wednesday of the month, because anytime is the right time for tea, starting at:
+* 7AM Pacific Time
+* 8AM Mountain Time,
+* 10AM Eastern Time,
+* 2PM UTC,
+* 4PM Central European Summer Time
+
+{{< button icon="fa calendar" spacing="true" color="info" tooltip="Download an ICS invite for the Kokkos tea-time serie" href="/kokkos-tea-time.ics" >}}
+Add to calendar
+{{< /button >}}
+
+<br>
+<br>
+
+The first international **Kokkos tea-time** organized by the  will take place on Wednesday, April the 17th, 2024 for 45min starting at 8AM MT, 10AM EST, 2PM UTC, 4PM CEST.
+It can be followed on [Zoom](https://cnrs.zoom.us/j/97834891802?pwd=NXhzd2paY051S3dQekVQVllvTW5MUT09), by [phone]({{< ref "#by-phone" >}}), from [a visio room]({{< ref "#from-a-visio-room" >}}) or from the [Mandelbrot room in the Digiteo Saclay building, France]({{< ref "#come-to-the-mandelbrot-room" >}}).
+
+Arnaud Durocher will give a talk entitled "Dyablo: A simulation code for astrophysics fluids with adaptive mesh refinement with Kokkos"
+
+
+## Dyablo: A simulation code for astrophysics fluids with adaptive mesh refinement with Kokkos
+
+*Arnaud Durocher, Maxime Delorme, CEA-Saclay, IRFU/DEDIP/LILAS*
+
+Dyablo is a new code for the simulation of astrophysical plasmas on exascale architectures using Adaptive Mesh Refinement. It is built from the ground up to target new and upcoming Exascale supercomputers with a new codebase in C++ using Kokkos. Dyablo is based on modern software engineering principles to allow "separation of concerns" and collaboration between HPC engineers and physicists through abstract interfaces and modularity. Although Dyablo is still in development, the code is now deployable and usable on all major (French) supercomputers.
+
+We will present Dyablo, how it's made and what challenges had to be resolved for AMR on GPUs. We will also present two physical problems we resolve with Dyablo in solar physics and cosmology.
+
+
+## Contribute
+
+Any subject is welcome if it is related to the Kokkos ecosystem (developments, usage, projects, issues, etc.) or even to GPU programming at large.
+Feel free to contact us to propose a talk: contact@cexa-project.org
+
+We target very informal discussion. You can propose any subject for discussion even if you don't have a dedicated slide deck.
+
+
+## Additional info
+
+### Join us physically in the Mandelbrot room
+
+The Mandelbrot room of [Digiteo Saclay building](https://maps.app.goo.gl/dWmfDSAyc6TLkrnC9) is in Saclay center, near the Orphée entrance.
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2631.8449189785!2d2.15407651328232!3d48.72755487119501!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47e67f3d6ca35351%3A0xad658a7f81b4a1fe!2sMaison%20de%20la%20Simulation!5e0!3m2!1sen!2sfr!4v1690451425962!5m2!1sen!2sfr" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+
```

Comment:
How about we use collapsible sections for these
```md
<details>
  <summary>In person</summary>
  The detail here
</details>
<details>
  <summary>Remote participation</summary>
  The detail here
</details>
```

Reply:
sounds great, I trieds `{{< colapsable ...` but it didn't work. That seems to be better

Reply:
Done, I tried to uncolapse the sections from the links at the beginning of the post, but didn't manage, I thinks its good like that, better publish before the event than publish a perfect post after the fact